earth_enterprise/src/server/geresetpgdb

```diff
@@ -83,8 +84,8 @@ else
 fi
 
 if [ $type == "hard" ]; then
-  echo "   WARNING: 'geresetpgdb $type' will completely delete postgresql data"
-  echo "directory, and then recreate the gestream, gesearch, gepoi, gesnippets"
+  echo "   WARNING: 'geresetpgdb $type' will completely delete all GEE postgresql"
```

Comment:
minor: all **Open** GEE ...

Reply:
Hmmm, technically the script would work with older version of GEE databases also. I'm inclined to leave off Open